### PR TITLE
Visualize all tract profiles for a scalar in one plot

### LINKED ISSUE #252: Visualize all tract profiles for a scalar in one plot
Something like this: 

```
tp = pd.read_csv(my_afq.get_tract_profiles()[0])

positions = OrderedDict({"ATR_L": (1, 0), "ATR_R": (1,4),  
                         "CST_L": (1,1), "CST_R": (1,3), 
                         "CGC_L": (3,1), "CGC_R": (3,3), 
                         "HCC_L": (4,1), "HCC_R": (4,3),  
                         "FP": (4, 2), "FA":(0,2), 
                         "IFO_L": (4, 0), "IFO_R": (4,4), 
                         "ILF_L":(3, 0), "ILF_R": (3,4), 
                         "SLF_L": (2, 1), "SLF_R": (2,3), 
                         "ARC_L": (2, 0), "ARC_R": (2,4), 
                         "UNC_L": (0, 1), "UNC_R": (0, 3)})

fig, axes = plt.subplots(5, 5)
afqb_colors = color_dict.values()


for ii, bundle in enumerate(positions.keys()):
    ax = axes[positions[bundle][0], positions[bundle][1]]
    fa = tp[(tp["bundle"] == bundle) & (tp["scalar"] == "dti_fa")]['profiles'].values
    ax.plot(fa, 'o-', color=afqb_colors[ii]) 
    ax.set_ylim([min_fa, max_fa])
    ax.set_yticks([0.2, 0.4, 0.6])
    ax.set_yticklabels([0.2, 0.4, 0.6])
    ax.set_xticklabels([])
    
fig.set_size_inches((12, 12))

axes[0, 0].axis("off")
axes[0, -1].axis("off")
axes[1, 2].axis("off")
axes[2, 2].axis("off")
axes[3, 2].axis("off")

```

That generates something like this: 

image

Could be a useful summary to provide routinely as part of the output.

### PR DESCRIPTION
**Pull request by 36000**
resolves #252

### CODE REVIEW

**Comment by 36000** [comment]
@arokem I believe this can be merged. Are the test suites working?

**Comment by arokem** [comment]
No. It looks like the new test you added is failing: https://github.com/yeatmanlab/pyAFQ/pull/268/checks?check_run_id=796766783

Does it pass locally for you on your machine?

**Comment by 36000** [comment]
It does pass locally. Could it be a problem using matplotlib in the test suite? I just pushed something that may help.

**Comment by arokem** [comment]
Let me know when you've rebased this one.

File: AFQ/viz.py [hunk 1]
```diff
@@ -15,6 +16,8 @@
 import AFQ.utils.volume as auv
 import AFQ.registration as reg
 
+import matplotlib
+matplotlib.use("Agg")
```
[the remark this answers is not in the document]
Comment (36000) [reply AFQ/viz.py:20]: I read online that this is necessary for matplotlib to run in a headless environment. However, upon further research, I am not sure this is helping. Let me push a commit without this backend and see if it still passes the tests.

[the remark this answers is not in the document]
Comment (36000) [reply AFQ/viz.py]: Turns out it was unnecessary. Good catch.

File: AFQ/viz.py [hunk 2]
```diff
@@ -15,10 +16,9 @@
 import AFQ.utils.volume as auv
 import AFQ.registration as reg
 
-
 tableau_20_rgb = np.array(Tableau_20.colors) / 255
 
-color_dict = OrderedDict({"ATR_L": tableau_20_rgb[0],
+COLOR_DICT = OrderedDict({"ATR_L": tableau_20_rgb[0],
```
Comment (arokem) [inline AFQ/viz.py:21]: 👍

**Pull request merged**